fineract-provider/src/main/java/org/apache/fineract/infrastructure/dataqueries/service/ReadWriteNonCoreDataServiceImpl.java

```diff
@@ -1940,8 +2047,8 @@ private String validateDatatableRegistered(String datatable) {
 
     private boolean isRegisteredDataTable(final String datatable) {
         final String sql = "SELECT COUNT(application_table_name) FROM " + TABLE_REGISTERED_TABLE + " WHERE registered_table_name = ?";
-        final int count = jdbcTemplate.queryForObject(sql, Integer.class, datatable);
-        return count > 0;
+        final Integer count = jdbcTemplate.queryForObject(sql, Integer.class, datatable);
```

Comment:
count function can never return null. using `int` is more than fine and no need for null check

Reply:
for me it was a checkstyle warning, if it can not return null, it is fine, but lets be safe